draft-ietf-quic-transport.md

```diff
@@ -5126,7 +5128,9 @@ approximately one in eight network paths.
 When the spin bit is disabled, endpoints MAY set the spin bit to any value, and
 MUST ignore any incoming value. It is RECOMMENDED that endpoints set the spin
 bit to a random value either chosen independently for each packet or chosen
-independently for each connection ID.
+independently for each connection ID. One way to accomplish this is to adjust
```

Comment:
This seems like a useful addition.

Reply:
No. No. No.  This does not work.  We discussed this.  It breaks the AEAD.